arm-templates/azuredeploy-ctm.json

```diff
@@ -90,12 +90,12 @@
     "keyVaultName": "[concat(parameters('serviceName'),'-ctm-kv',variables('suffix'))]",
     "ayalonWebAppName": "[concat(parameters('serviceName'),'-ayalon-webapp',variables('suffix'))]",
     "managedIdentityName": "[concat(parameters('serviceName'),'-managed-identity',variables('suffix'))]",
+    "ayalonServicePlanName": "[concat(parameters('serviceName'),'-ayalon-plan',variables('suffix'))]",
```

Comment:
where do you add the actual resource?

Reply:
it is added in BP, just the name is changed when secondary. which will make 2 instances deployed